editor/src/clj/editor/targets.clj

```diff
@@ -221,8 +226,8 @@
                   (filter some?))
         errors (filter string? targets-result)
         {external-targets false local-targets true} (group-by local-target? targets)
-        targets (into [] (comp cat (distinct)) [(sort-by :name util/natural-order local-targets)
-                                                (sort-by :name util/natural-order external-targets)])]
+        targets (into [] (comp cat (distinct)) [(sort-by :url local-targets)
+                                                (sort-by :url external-targets)])]
```

Comment:
Names do not contain the port, so if we have a few instances the names for them will be the same, which might be a reason of infinite updating of the menu (I caught it many times)